File: src/types/sentiment.ts
export type SentimentVariant = "hot" | "risky";

export interface SentimentItem {
  name: string;
  score: number;
  description: string;
}

export interface SentimentData {
  hotSectors: SentimentItem[];
  riskySectors: SentimentItem[];
  hotRegions: SentimentItem[];
  riskyRegions: SentimentItem[];
}

File: backend/app/schemas/sentiment.py
from pydantic import BaseModel


class SentimentItem(BaseModel):
    name: str
    score: float
    description: str


class SentimentResponse(BaseModel):
    hot_sectors: list[SentimentItem]
    risky_sectors: list[SentimentItem]
    hot_regions: list[SentimentItem]
    risky_regions: list[SentimentItem]
